app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -938,9 +938,12 @@ def build_toolbar_disable_button(id)
     return true if ["button_add", "button_save", "button_reset"].include?(id) && !@changed
 
     # need to add this here, since this button is on list view screen
-    if @layout == "pxe" && id == "iso_datastore_new" && ManageIQ::Providers::Redhat::InfraManager.datastore?
-      return N_("No %{providers} are available to create an ISO Datastore on") %
-        {:providers => ui_lookup(:tables => "ext_management_system")}
+    if @layout == "pxe" &&
+       id == "iso_datastore_new" &&
```

Comment:
If this is going to be broken into multiple lines, I think this would be easier to read if the `&&`s were replaced with `if`s